src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -439,27 +478,29 @@ private function maybe_organizer_is_separated_list( $organizer ) {
 	}
 
 	/**
-	 * Handle finding the matching organizer(s) for the event
-	 * @since 4.6.19
-	 * @param $record - the event record from the import
+	 * Handle finding the matching organizer(s) for the event.
 	 *
-	 * @return array
+	 * @since 3.2.0
+	 *
+	 * @param array<string|mixed> $record The event record from the import.
```

Comment:
What do you mean here? 
* "Array with string keys and mixed values"? It's `array<string,mixed>`
* "Array with values that can be either strings or mixed? It's just `array<mixed>` since `mixed` contains the `string` type.

Reply:
@lucatume It seems to me it's an array with string keys and mixed values.
If that is the case, all instances of `array<string|mixed> $record` should be updated in the file I guess.

Reply:
Yes, they should be updated if you are sure they are  in fact arrays of that shape.

Reply:
@lucatume Updated all of them.